pkg/pcr/measurement.go

```diff
@@ -120,6 +120,9 @@ type Measurement struct {
 
 	// Data contains chunks of data to be measured as contiguous sequence of bytes
 	Data DataChunks `json:",omitempty"`
+
+	// NoHashing tells that the data of this measurement should not be hashed and should be treated as hash
+	NoHashing bool
```

Comment:
This is not needed. See method `IsFake`. You just need a new MeasurementID.

Reply:
IsFake is different, it skips the measurement and I need a way to skip hashing of this value

Reply:
Yeah, I've mistaken.

Reply:
In this case let's move `NoHash` to a method? I see no reason to keep it as a variable.